### PR TITLE
[TOREE-554] Use Pekko instead of Akka

### PR DESCRIPTION
**Pull request by pjfanning**
https://issues.apache.org/jira/browse/TOREE-554

### CODE REVIEW

**Comment by lresende** [comment]
Thank you, let me do some testing and let you know, otherwise code changes lgtm

File: project/Dependencies.scala [hunk 1]
```diff
@@ -21,16 +21,16 @@ import scala.util.Properties
 object Dependencies {
 
   // Libraries
-  val akkaVersion = "2.6.21" // The latest version under Apache v2
-  val akkaActor = "com.typesafe.akka" %% "akka-actor" % akkaVersion // Apache v2
-  val akkaSlf4j = "com.typesafe.akka" %% "akka-slf4j" % akkaVersion // Apache v2
-  val akkaTestkit = "com.typesafe.akka" %% "akka-testkit" % akkaVersion // Apache v2
+  val pekkoVersion = "1.0.2" // The latest version under Apache v2
```
Comment (pan3793) [inline project/Dependencies.scala:24]: comment is outdated
```suggestion
  val pekkoVersion = "1.0.2" // Apache v2
```

**Comment by lresende** [comment]
rerun build

**Comment by lresende** [comment]
@pjfanning I finally had time to test this in some of my workloads and it seem to be ok, could you please try to re-trigger the pr ci/cd to get a green test build

**Comment by pjfanning** [comment]
@lresende the tests have passed on most recent run

**Pull request merged**